test/unit/ItemFilter.test.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-env mocha */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import { spy } from 'sinon';
+
+import ItemFilter from './../../src/app/components/Item/ItemFilter';
+import { locationFilters, statusFilters } from '../fixtures/itemFilterOptions';
+
+describe('ItemFilters', () => {
+  describe('default rendering', () => {
+    let component;
+    it('should not render without an `items` prop', () => {
+      component = shallow(<ItemFilter />);
+      expect(component.type()).to.equal(null);
+    });
+  });
+
+  describe('with `options`', () => {
+    let component;
+    it('should render a `div` and a `button`', () => {
+      component = mount(<ItemFilter options={locationFilters} />);
+      expect(component.find('div').length).to.equal(1);
+      expect(component.find('button').length).to.equal(1);
+    });
+  });
+
+  describe('with `options` open state', () => {
+    let component;
+    it('should render a fieldset, 3 buttons', () => {
+      component = mount(
+        <ItemFilter
+          options={locationFilters}
+          isOpen
+        />);
+      expect(component.find('fieldset').length).to.equal(1);
+      expect(component.find('button').length).to.equal(3);
+    });
+  });
+
+  describe('with `selectedFilters`', () => {
+    it('clear button should remove selected filters for corresponding `filter`', () => {
```

Comment:
@nypl-wluisi here's the test I wrote for the "Clear" button. It just checks that the right filters are removed. It is very tied to the implementation here in which `submitFilterSelections` is passed from the parent component.